nexum_core/BENCHMARKS.md

```
@@ -0,0 +1,226 @@
+cargo bench
+
+# Run specific benchmark suite
+cargo bench --bench storage_bench
+
+# Use the benchmark runner script (Linux/macOS)
+./run_benchmarks.sh
+```
+
+## Benchmark Suites
+
+### 1. Storage Engine (`storage_bench.rs`)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)